portfolio/src/main/webapp/script.js

```diff
@@ -53,16 +53,27 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-// Creates a map and adds it to the page.
+// Creates a map and adds it to the page with markers on compost-dropoff locations.
 function initMap() {
-  // Coordinates center map  around Baltimore, MD
-  var myLatlng = new google.maps.LatLng(39.412, -76,775);
+  // Coordinates centers map  around NYC
+  var myLatlng = {lat: 40.763232, lng: -73.951047};
   var mapOptions = {
     center: myLatlng, 
-    zoom: 12, 
-    mapTypeId: 'roadmap'
+    zoom: 12,
+    mapTypeControl: false 
   };
   const map = new google.maps.Map(document.getElementById('map'), mapOptions);
-}
-
+  var image = 'http://maps.google.com/mapfiles/kml/pal4/icon47.png';
```

Comment:
It's a good practice to define this as a constant, I imagine the're is no need for it to change at runtime.